test/integration/voice-chat/community-voice-chat-expiration-job.spec.ts

```diff
@@ -0,0 +1,146 @@
+import { test } from '../../components'
+import { getCommunityVoiceChatRoomName } from '../../../src/logic/voice/utils'
+import { VoiceChatUserStatus } from '../../../src/adapters/db/types'
+
+test('Community voice chat expiration job', ({ components }) => {
+  const communityId = 'test-community-expiration'
+  const roomName = getCommunityVoiceChatRoomName(communityId)
+  const moderatorAddress = '0x1234567890123456789012345678901234567890'
+  const memberAddress = '0x1234567890123456789012345678901234567891'
+
+  beforeEach(async () => {
+    // Clean up any existing test data
+    try {
+      await components.voiceDB.deleteCommunityVoiceChat(roomName)
+    } catch (error) {
+      // Ignore if room doesn't exist
+    }
+  })
+
+  afterEach(async () => {
+    try {
+      await components.voiceDB.deleteCommunityVoiceChat(roomName)
+    } catch (error) {
+      // Ignore if room doesn't exist
+    }
+  })
+
+  describe('when a community room has no active moderators for more than 5 minutes', () => {
+    it('should be deleted by the expiration job', async () => {
+      // Create a community room with a moderator
+      await components.voiceDB.joinUserToCommunityRoom(moderatorAddress, roomName, true)
+      
+      // Add a member
+      await components.voiceDB.joinUserToCommunityRoom(memberAddress, roomName, false)
+      
+      // Verify room exists
+      const usersInRoom = await components.voiceDB.getCommunityUsersInRoom(roomName)
+      expect(usersInRoom).toHaveLength(2)
+      
+      // Simulate moderator disconnecting (but not leaving voluntarily)
+      await components.voiceDB.updateCommunityUserStatus(
+        moderatorAddress, 
+        roomName, 
+        VoiceChatUserStatus.ConnectionInterrupted
+      )
+      
+      // Move the moderator's disconnect time to 6 minutes ago (simulate time passing)
+      const sixMinutesAgo = Date.now() - (6 * 60 * 1000)
+      await components.database.query(
```

Comment:
As the process of updating the status of the `community_voice_chat_users` is done twice, what do you think about creating a helper to abstract this like the ones we have in `db-utils.ts`?